SUMMARY.md

```diff
@@ -50,6 +50,7 @@
   * [Azure Cloud Storage](./content/chapter-03/AzureCloudStorage.md)
   * [Oracle Cloud Storage](./content/chapter-03/OracleCloudStorage.md)
 * [Intermittent connectivity issues](./content/chapter-03/IntermittentConnectivity.md)
+* [Resilience of software design](./content/chapter-03/Resilience.md)
```

Comment:
Resilience of software design => Resilient Software Design

Reply:
Done